pyproject.toml

```diff
@@ -64,6 +64,7 @@ hsaf_grib = ["pygrib"]
 remote_reading = ["fsspec"]
 insat_3d = ["xarray>=2024.10.0"]
 gms5-vissr_l1b = ["numba"]
+aws_l1b = ["xarray-datatree"]
```

Comment:
Just a check, but isn't this now a part of `xarray` and doesn't need a separate dependecy? So maybe make the reader dependent on a recent `xarray` release.

Reply:
Yes, good catch!